new/LIT5923TenCommand.xml

```diff
@@ -0,0 +1,56 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT5923TenCommand" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ነገረ፡ በእንተ፡ አሠርቱ፡ ቃላት፡</title>
```

Comment:
it should be nagar, is it nagara in the manuscript?

Reply:
This is the way Sophia catalogued in SDM-021

Reply:
I just checked, the manuscript has nagar, then we can correct it here

Reply:
Checked in MS, it is nagar sure enough, thank you